docs/topics/migrations.txt

```diff
@@ -224,14 +224,14 @@ Swappable dependencies
 .. function:: django.db.migrations.swappable_dependency(value)
 
 The ``swappable_dependency()`` function is used in migrations to declare
-"swappable" dependencies on migrations in the app of the swapped-in model.
-Currently, the first migration of this app and as a consequence the swapped-in
+"swappable" dependencies on migrations in the app of the swapped-in model,
+currently, the first migration of this app. As a consequence, the swapped-in
```

Comment:
@smithdc1 I added "on", is it still good? :thinking:

Reply:
I think so.

I'd read "part of" or "on" as the same meaning here. 

Does that help?